intermediate_html_css/intermediate_css_concepts/css_units.md

```diff
@@ -1,14 +1,14 @@
 ### Introduction
 There are many different units that you can use to define sizes in CSS. This lesson will introduce the most important to you, and show you where to learn about the rest of them.
 
-### Learning Outcomes
+### Learning outcomes
```

Comment:
```suggestion
### Learning outcomes

```

Reply:
These headers seem to be missing a newline below them so I decided to fix them while I'm at it. Feel free to `commit suggestion` these if you approve